backend/app/services/implementations/log_records_service.py

```diff
@@ -214,8 +233,14 @@ def count_log_records(self, filters=None):
             COUNT(*)\n \
             FROM log_records logs\n \
             LEFT JOIN users attn_tos ON logs.attn_to = attn_tos.id\n \
-            JOIN users employees ON logs.employee_id = employees.id"
-            
+            JOIN users employees ON logs.employee_id = employees.id\n\
+            LEFT JOIN\n \
```

Comment:
Then you can just call the helper function here, similar to `join_tag_attributes`